### PR TITLE
Allow editing motif areas in scrolled editor

### PR DESCRIPTION
**Pull request by tf**
REDMINE-17343

### CODE REVIEW

File: entry_types/scrolled/config/locales/new/de.yml [hunk 1]
```diff
@@ -219,6 +219,7 @@ de:
           Dieser Schritt kann nicht rückgängig gemacht werden.
         save_error: Beim Speichern des Kapitels ist ein Fehler aufgetreten.
       edit_section:
+        edit_motif_area: Bildmotif markieren...
```
Comment (DanielKuhn) [inline entry_types/scrolled/config/locales/new/de.yml:222]: "Motif" (english) => "Motiv" (german)

File: entry_types/scrolled/config/locales/new/de.yml [hunk 2]
```diff
@@ -290,3 +291,10 @@ de:
       insert_content_element:
         header: Element einfügen
         cancel: Abbrechen
+      edit_motif_area:
+        header: Bildmotif
```
Comment (DanielKuhn) [inline entry_types/scrolled/config/locales/new/de.yml:295]: "Bildmotiv" as well

File: entry_types/scrolled/package/src/editor/vendor/jquery.imgareaselect.module.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+:global .imgareaselect-handle {
+  background-color: #fff;
+  border: solid 1px #000;
+  opacity: 0.5;
+}
+
+:global .imgareaselect-outer {
+  background-color: #000;
+  opacity: 0.8;
```
Comment (DanielKuhn) [inline entry_types/scrolled/package/src/editor/vendor/jquery.imgareaselect.module.css:16]: I think the outer area is currently too dark. Especially in blank slate the image is hardly visible. Here an opacity of 0.5 would suffice imo.

Comment (tf) [reply entry_types/scrolled/package/src/editor/vendor/jquery.imgareaselect.module.css]: Changed.

**Pull request merged**